packages/components/time-picker/src/time-picker-com/basic-time-spinner.vue

```diff
@@ -233,11 +233,38 @@ const scrollDown = (step: number) => {
   const total = currentScrollbar.value === 'hours' ? 24 : 60
   now = (now + step + total) % total
 
+  if (props.arrowControl) {
```

Comment:
Maybe it should not be restricted to only `arrowControl`, it should be the default behavior.

eg, There are still problems with using the arrow keys in the keyboard